src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
Use more descriptive name then obj, com'on Lera, what a naming is that!👀

Reply:
what's wrong with the name? We iterate through objects in actions. What can be more clear?

Reply:
We are humans, and as a human It makes more sense to us when we say we iterate over **single** action in actions 🙌

Be more abstract, for your own sake!